monkestation/code/modules/virology/disease/symtoms/stage3.dm

```diff
@@ -720,34 +720,29 @@ GLOBAL_LIST_INIT(disease_hivemind_users, list())
 			Heal(mob, multiplier)
 		else
 			multiplier = min(multiplier + 0.1, max_multiplier)
-	return
 
 /datum/symptom/darkness/proc/CanHeal(mob/living/carbon/mob)
 	var/light_amount = 0
 	if(isturf(mob.loc)) //else, there's considered to be no light
-		var/turf/T = mob.loc
-		light_amount = min(1,T.get_lumcount()) - 0.5
+		var/turf/mob_turf = mob
```

Comment:
why are you setting turf to the mob?

Reply:
![bleh](https://github.com/user-attachments/assets/252e2111-e340-4306-9b49-a876bc851df5)